rpc_jobs/rpc_gating_project_builds.yml

```diff
@@ -0,0 +1,29 @@
+- job:
+    name: 'RPC-Gating-Project-Builds'
+    project-type: workflow
+    concurrent: true
+    triggers:
+      - github-pull-request:
+          org-list:
+            - rcbops
+          github-hooks: true
+          trigger-phrase: '.*recheck_all.*|.*recheck_project_builds.*'
```

Comment:
When I have to recheck, I look at the PR checks in github, and replace / or - with _. It generally works.

Here it won't, because recheck_cit_project_builds won't trigger.
But it's details.